docs/writing-an-intent.md

```diff
@@ -178,3 +178,15 @@ Use `log.info()` (debug, etc) for any logging you plan to leave in the code. `lo
 If you are doing debugging, use `console.log()`. These are _not_ allowed in the code (and `npm test` will fail), but that's intentional: you should remove any debugging before committing the code, and only leave deliberate log messages.
 
 If you want to see the inter-process communication, run `LOG_LEVEL=messaging npm start`.
+
+## NPM scripts
+
+NPM scripts are simply terminal commands. The npm scripts which are available to use are listed in package.json file. `enumerate-phrases` is a project specific/custom npm script.
+
+**enumerate-phrases:** lists all the matched phrases for the specified intent using the auto-generated metadata for finding matches. This metadata is generated from all the .toml files containing intents description. 
+
+You can execute this script as:
+```js 
+npm run enumerate-phrases --someIntent.command
+```
+For example, Running the command `npm run enumerate-phrases --navigation.bangSearch` lists all the matched phrases for navigation.bangSearch intent.
```

Comment:
Same here – the command you give returns every possible phrase (all 76k of them!)